client/battle/BattleInterfaceClasses.cpp

```diff
@@ -483,7 +483,8 @@ void QuickSpellPanel::create()
 		{
 			buttonsDisabled.push_back(std::make_shared<TransparentFilledRectangle>(Rect(2, 7 + 50 * i, 48, 36), ColorRGBA(0, 0, 0, 172)));
 		}
-		labels.push_back(std::make_shared<CLabel>(7, 10 + 50 * i, EFonts::FONT_TINY, ETextAlignment::TOPLEFT, Colors::WHITE, config["keyboard"]["battleSpellShortcut" + std::to_string(i)].String()));
+		if(GH.input().getCurrentInputMode() == InputMode::MOUSE)
```

Comment:
You're actually checking for keyboard here, not for mouse.
What if player connects external mouse to his phone? Or an opposite case - external keyboard?

Although not sure what's the proper solution here. Might be better to leave as it, at least until we actually run into such issue.

Reply:
Yeah, I know. Currently I assume Keyboard and Mouse are combined. Should be 99.9% of the playerbase.

Theoretically it would be also possible to play with touch and keyboard (or gameplay and mouse 🤣).

Only disadvantage currently is using mouse and touch combined showing the numbers.

But no idea how to improve. Would leave as it is for now.

Reply:
In that case - let's just rename this input mode to KEYBOARD_AND_MOUSE or something like that.

Current code looks a bit weird even without accounting for these edge cases - we check for MOUSE input mode and proceed to show keyboard hotkeys.

Reply:
Yes, sure. Done. :)